recipes-domx/meta-xt-images-vgpu/recipes-core/process-cpp/process-cpp_2.0.0.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "A simple convenience library for handling processes in C++11."
+HOMEPAGE = "https://launchpad.net/process-cpp"
+
+LICENSE = "LGPLv3"
+LIC_FILES_CHKSUM = "file://COPYING;md5=e6a600fd5e1d9cbde2d983680233ad02"
+DEPENDS = "boost \
+           gtest \
+           gmock \
+           properties-cpp \
+"
+
+SRC_URI = "bzr://bazaar.launchpad.net/~phablet-team/process-cpp/trunk"
+SRCREV = "54"
```

Comment:
why this rev?

Reply:
All revisions mentioned above are latest/stable. With "norev" or ${AUTOREV} bzt fetches the oldest one.

Reply:
got it